_posts/2022-03-07-using-private-image-repositories.markdown

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title:  "Deploying workloads with images from private repositories"
+date:   2022-02-07 17:00:00 +0100
+categories: flotta
+author: Jakub Dżon
+tags:
+  - guide
+  - flotta
+  
+summary-1: Deploying workloads with images from private repositories
+---
+
+Users can deploy their containerized workloads to edge devices using Flotta and very often images they want to use are 
+not public and are available only for users or services having proper credentials. 
+
+Flotta supports that use case and allows users to use private images in their `EdgeDeployments`; this post shows how to 
+make use of that support by walking you through all the required steps.
+
+### Prerequisites
+
+1. `podman` installed
+2. Private image repository (quay.io, for example)
+3. Image of the workload to deploy
+4. Running `EdgeDevice` labelled with `dc=home` 
+ 
+#### Image of the workload to deploy
+
+For the purpose of this walk-through we will use nginx server image (docker.io/nginx:1.14.2) as the one that would be put 
+in a private quay.io repository. You can, of course, use any private container image repository.
+
+1. Login to quay.io:
+```bash
+podman login quay.io
+```
+You will be prompted for your username and password:
+```bash
+$ podman login quay.io                                                                                                     
+Username: xxxxxx
+Password: ******
+Login Succeeded!
+```
+
+1. Pull `docker.io/nginx:1.14.2`
+```bash
+podman pull docker.io/nginx:1.14.2
+```
+
+1. Tag `docker.io/nginx:1.14.2` with a private name
```

Comment:
Did you check if numbered list must be progressive or not (e.g. `1.` `2.` `3.` ... )?

Reply:
It doesn't have to be. Checked in local jekyll.